fem/conduitdatacollection.hpp

```diff
@@ -209,6 +209,9 @@ class ConduitDataCollection : public DataCollection
                                                      const conduit::Node &n_field,
                                                      bool zero_copy = false);
 
+   /** Checks if any rank in the mesh has boundary elements */
+   static bool HasBoundaryElements(mfem::Mesh *mesh);
```

Comment:
Minor: Since the `ConduitDataCollection` is over a specific mesh, perhaps this should be a non-static member function?